trajectorytools/trajectories/concatenate.py

```diff
@@ -64,14 +66,54 @@ def _concatenate_idtrackerai_dicts(traj_dicts):
     return traj_dict_cat
 
 
-def from_several_idtracker_files(trajectories_paths, **kwargs):
+def pick_trajectory_file(session_folder):
+    """Select the best trajectories file
+    available in an idtrackerai session
+    """
+    trajectories = os.path.join(session_folder, "trajectories", "trajectories.npy")
+    trajectories_wo_gaps = os.path.join(session_folder, "trajectories_wo_gaps", "trajectories_wo_gaps.npy")
+
+    if os.path.exists(trajectories_wo_gaps):
+        return trajectories_wo_gaps
+    elif os.path.exists(trajectories):
+        return trajectories
+    else:
+        raise Exception(f"Session {session_folder} has no trajectories")
+
+
+def is_idtrackerai_session(path):
```

Comment:
I believe it would be more robust to check if it contains a file called `video_object.npy`. 
the pattern `session_` could be also used for sessions of experiments and you can end up checking for trajectories in folders that are not needed.